app/uk/gov/hmrc/transitmovements/models/Movement.scala

```diff
@@ -16,17 +16,15 @@
 
 package uk.gov.hmrc.transitmovements.models
 
-import cats.data.NonEmptyList
-
 import java.time.OffsetDateTime
 
 case class Movement(
   _id: MovementId,
   movementType: MovementType,
   enrollmentEORINumber: EORINumber,
-  movementEORINumber: EORINumber,
+  movementEORINumber: Option[EORINumber],
   movementReferenceNumber: Option[MovementReferenceNumber], // optional pending MRN allocation
   created: OffsetDateTime,
   updated: OffsetDateTime,
-  messages: NonEmptyList[Message]
+  messages: Vector[Message]
```

Comment:
Why a `Vector`? `Seq` should suffice -- we should let the runtime decide what's best unless we specifically need something from a more specialised type.